apps/omg_api/lib/application.ex

```diff
@@ -68,7 +68,7 @@ defmodule OMG.API.Application do
       {OMG.RPC.Web.Endpoint, []}
     ]
 
-    opts = [strategy: :one_for_one]
+    opts = [strategy: :one_for_one, name: __MODULE__, max_restarts: 10000, max_seconds: 60_000]
```

Comment:
What does `max_seconds` represent? The interval between restarts?

Reply:
The combo max restarts and max seconds specify maximum restart intensity.
If more than `max_restarts` number of restarts occur in the last `max_seconds` seconds, the supervisor terminates all the child processes and then itself, ending in terminating `omg_api` application.